GCM/App/editor/pointOfInterest/PointOfInterestController.java

```diff
@@ -0,0 +1,93 @@
+package editor.pointOfInterest;
+
+import static java.lang.Math.toIntExact;
+
+import java.net.URL;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.ResourceBundle;
+
+import dataAccess.users.PurchaseDetails;
+import gcmDataAccess.GcmDAO;
+import javafx.event.EventHandler;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.input.MouseEvent;
+import maps.Coordinates;
+import maps.Site;
+
+public class PointOfInterestController implements Initializable{
+	private GcmDAO gcmDAO;
+	
+	@FXML
+	TextField name;
+	@FXML
+	TextField xC;
+	@FXML
+	TextField yC;
+	@FXML
+	TextField type;
+	@FXML
+	TextField description;
+	@FXML
+	RadioButton disableYes;
+	@FXML
+	RadioButton disableNo;
+	@FXML
+	Button addEdit;
+	@FXML
+	public ToggleGroup disableOptions;
+	
+	String selectedRadioBtn;
+	RadioButton selectRadio;
+	
+	int mapId;
+	
+	public PointOfInterestController(GcmDAO gcmDAO, int mapId) {
```

Comment:
This constructor takes a "**`mapId`**"
In the bottom it feeds the `mapId `to the `Site `constructor in the argument that takes **`siteId`**, and then calls `addNewSiteToCity` with the `mapId` as an argument for **`cityId`**